packages/api-nest/src/highlight/entities/highlight.entity.ts

```
@@ -0,0 +1,96 @@
+  createdAt!: Date
+
+  @UpdateDateColumn({ name: 'updated_at' })
+  updatedAt!: Date
+
+  @Column({ name: 'shared_at', type: 'timestamptz', nullable: true })
+  sharedAt?: Date | null
+
+  @Column({ name: 'highlight_position_percent', type: 'real', default: 0 })
+  highlightPositionPercent!: number
+
+  @Column({ name: 'highlight_position_anchor_index', type: 'integer', default: 0 })
```

ESLint

[error] Replace `·name:·'highlight_position_anchor_index',·type:·'integer',·default:·0` with `⏎····name:·'highlight_position_anchor_index',⏎····type:·'integer',⏎····default:·0,⏎·`

(prettier/prettier)